_content/essays/love-unknowing-and-female-filth_amy_paris_langenberg.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Love, Unknowing, and Female Filth: The Buddhist Discourse of Birth as a Vector of Social Change for Monastic Women in Premodern South Asia"
+authors:
+  - "Amy Paris Langenberg"
+external_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/pdf"
+source_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/html"
+drive_links:
+  - "https://drive.google.com/file/d/1vh9BaxP8ORYmgD1FBIRTvjvWAJe2IQJe/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
+  - gender
+  - vinaya
+  - nun
+year: 2021
+publisher: degruyter
+address: "Germany"
+from_book: Primary Sources and Asian Pasts
+pages: "308--340"
+minutes: 
+---
+This artcile argues that "the Buddhist tale of the impure, disgusting, and violent female body and the suffering of the fetus within the womb, so seemingly negative toward women, in fact operated discursively and affectively to support premodern female Buddhist monasticism by helping to generate a moral-social
```

Comment:
Instead of “This artcile [sic] argues that "the Buddhist…” please just use the block quote feature:

```md
> the Buddhist…
```